recipes/conda-devenv/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "conda-devenv" %}
+{% set version = "0.9.1" %}
+{% set hash = "fe8e9be7d4f2df03b9a269a4b4b36ca67477c9b1d5c2af1af440f63b787f2f05" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: '{{ version }}.tar.gz'
```

Comment:
Optional, but could structure your filename like [this]( https://github.com/conda-forge/staged-recipes/blob/0394a09dfc8024382222d1cb77a523d7647734e3/recipes/example/meta.yaml#L17 ). Helps with caching (and finding things 😄) if one does a lot of builds locally.